source/copilot-docs.txt

```diff
@@ -0,0 +1,69 @@
+.. _vsce-copilot-docs:
+
+==============
+/docs Command
+==============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The ``/docs`` command provides MongoDB-specific information,
+supplemented by links to MongoDB Documentation. The |copilot| is
+knowledgeable of the latest MongoDB Documentation and uses the
+MongoDB Docs Chatbot to link to information in addition to
+the text response. 
+
+Example
```

Comment:
Is it possible to add a second example here where the docs command recommends a DDP product like vector search or search in the context of "how do I improve the searching speed for <this> query" and it responds with search/vector search recommendation? For vector search we might have to ask something like 'semantic search' in the prompt. This way users know that it can also be leveraged for performance-related questions.

Reply:
Added a second example that recommends Atlas Vector Search!